code/modules/modular_computers/file_system/programs/app_presets.dm

```diff
@@ -51,6 +51,7 @@
 		new /datum/computer_file/program/filemanager(),
 		new /datum/computer_file/program/chatclient(),
 		new /datum/computer_file/program/civilian/cargoorder(),
+		new /datum/computer_file/program/comm(TRUE),
```

Comment:
Change all these to FALSE and look for other instances where this defaults to true for mobile devices.
The first parameter is used to determine if command reports should be printed.
So if you put the laptop in your backpack you leave reports wherever you go (as they are spawned on the tile below the computer; Laptops were never meant to have printers)